MVP Module/Source/Feed UI/Composers/FeedViewAdapter.swift

```diff
@@ -5,7 +5,7 @@
 import UIKit
 import FeedFeature
 
-final class FeedViewAdapter: FeedView {
+final class FeedViewAdapter: FeedViewFlow {
```

Comment:
Following from my previous comments, we can leave that adapter responsible for adapting the `Feed` display, and leaving `errorMessages` to be displayed directly by the `FeedViewController`, avoiding another layer of indirection.
```suggestion
final class FeedViewAdapter: FeedView {
```

Reply:
done